### PR TITLE
Created Club Profile Page

### PR DESCRIPTION
**Pull request by laurasofi1024**
Established Club Profile Page

Created Component for the Main Content of the Profile Page

### CODE REVIEW

**Comment by laurasofi1024** [comment]
Current Desktop View:
Screen Shot 2020-11-13 at 4 23 51 PM

**Comment by laurasofi1024** [comment]
@erin2722  I made the application link a button. This was improvised so please let me know if I should change it back to a link or style it differently

**Review by erin2722 — CHANGES_REQUESTED** [review]
This looks incredible!

I think that looking at Francesca's explore page branch will help clarify a lot of the questions you sent, and I hope to get that merged in quite soon. The basic infrastructure that you have commented out is exactly right. 🎉 👏 

For now, maybe add in the meeting frequency box, the extra button, and the tablet view, and then once francesca's PR is merged in we will get this working even better 🥇

**Comment by laurasofi1024** [comment]
Changes made:
- Added Join mailing list button
- Created Tablet View
- Changed location of apply button per the updated Figma mockup
- Made changes to margins of FrequencyTag and ProfilePageBox components

General comments:
- I assume there will be a specific link for the mailing list added to the database later on so I just made its button's link `club.mailingList` for now
- The FrequencyTag component requires a string for frequency (i.e "1", "2", "3" etc) and a boolean for weekly. I put `club.meetingFrequency` and `club.weekly` for each of these respectively for now pending how these will be represented in the database. 
- How can I access the database for my local app? I assume I have to download MongoDB locally or something along those lines

**Comment by laurasofi1024** [comment]
Current Desktop View:
Screen Shot 2020-11-28 at 10 51 41 PM

Current Mobile View:
localhost_3000_clubs(iPhone X)

Current Tablet View:
localhost_3000_clubs(iPad)

File: client/src/components/profileMainContent/index.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faCheckCircle } from "@fortawesome/free-regular-svg-icons";
+
+export const ProfileMain = ({description, highlights, howToJoin}) => {   
+    return (
+        <Container>
+            <h2>Description</h2>
+            <p className='one'>{description}</p>
+            <h2>Highlights</h2>
+            <p className='two'>{highlights.map(highlight => <Highlight text={highlight}/>)}</p>
```
Comment (erin2722) [inline client/src/components/profileMainContent/index.js:12]: this looks great!

File: client/src/containers/ClubProfile.js [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+            })
+          .then((res) => res.json())
+          .then((response) => {
+            setClub(response);
+            setLoading(false);
+            console.log(`api/clubProfiles/${id}`);
+          })
+          .catch((error) => console.log(error));
+    }, [id]);
+
```
Comment (erin2722) [inline client/src/containers/ClubProfile.js:30]: I would recommend adding this variable and then replacing all instances of the similar clubs code with `<SimilarClubs />` in order to make the code shorter/more readable
```suggestion
        const SimilarClubs = () => { return (club.similarClubs.length === 0) ? (<h1>Loading</h1>) : 
        (club.similarClubs.map(profile => (
            <ExploreBox 
                name = {profile.name}
                description = {profile.shortDescription}
                imageURL = {profile.imageUrl}
                tags = {profile.tags}
                clubSize = {profile.memberRange}
                acceptingMembers = {profile.acceptingMembers}
                applicationRequired = {profile.applicationRequired}
                cardLink={`/clubs/${profile._id}`}
            />
    )))};
```

File: client/src/containers/ClubProfile.js [hunk 3]
```diff
@@ -0,0 +1,264 @@
...
+                                        name = {profile.name}
+                                        description = {profile.shortDescription}
+                                        imageURL = {profile.imageUrl}
+                                        tags = {profile.tags}
+                                        clubSize = {profile.memberRange}
+                                        acceptingMembers = {profile.acceptingMembers}
+                                        applicationRequired = {profile.applicationRequired}
+                                        cardLink={`/clubs/${profile._id}`}
+                                    />
+                                )))}
```
Comment (erin2722) [inline client/src/containers/ClubProfile.js:71]:
```suggestion
<SimilarClubs />
```

**Review by erin2722 — CHANGES_REQUESTED** [review]
🎉 this looks incredible! 

After the small changes we will get this merged in!

**Review by erin2722 — APPROVED** [review]

**Pull request merged**